riptide-spring-boot-autoconfigure/README.md

```diff
@@ -575,16 +575,25 @@ For every client that is defined in your configuration the following beans will
 [![Client Dependency Graph](../docs/interceptors.png)](../docs/interceptors.png)
 
 Every single bean in the graph can optionally be replaced by your own, custom version of it. Beans can only be
-overridden by name, **not** by type. As an example, the following code would add XML support to the `example` client:
+overridden by name, **not** by type. You can define bean name by using an appropriate method name or specify it explicitly via `@Bean(name = "...")`,
+you cannot use `@Qualifier("...")` annotation since it doesn't change an actual bean name.
+As an example, the following code would add XML support to the `example` client:
 
 ```java
 @Bean
-@Qualifier("example")
 public ClientHttpMessageConverters exampleHttpMessageConverters() {
     return new ClientHttpMessageConverters(singletonList(new Jaxb2RootElementHttpMessageConverter()));
 }
 ```
 
+The following code can be used if you cannot use your client name in the method name (e.g. your client name is `my-client`):
+@Bean(name = "my-clientCircuitBreakerExecutorService")
```

Comment:
@Semernitskaya you need to open code block before this code snippet

Reply:
Fixed